spyder/plugins/variableexplorer/widgets/main_widget.py

```diff
@@ -440,13 +370,87 @@ def _setup(self):
         """
         Create options menu and adjacent toolbar buttons, etc.
 
+        This creates base actions related with Search, Filter and Refresh.
+
         This calls the parent's method to setup default actions, create the
         spinner and the options menu, and connect signals. After that, it adds
         the Search, Filter and Refresh buttons between the spinner and the
         options menu.
         """
         super()._setup()
 
+        # ---- Base Options menu actions
+        self.exclude_private_action = self.create_action(
```

Comment:
Why was it necessary to move these actions here? Just curious.

Reply:
Do to the change done at https://github.com/spyder-ide/spyder/pull/21353/files#diff-408fe471334217d5c333ce2fd096e3f8369ce9495549dbdae1fe1038435a9a4c